src/Confluent.RestClient/IConfluentClientSettings.cs

```diff
@@ -3,5 +3,6 @@
     public interface IConfluentClientSettings
     {
         string KafkaBaseUrl { get; }
+        long RequestTimeout { get; }
```

Comment:
Hi @inadler, thanks for the changes.

This confluent client is actually a HTTP client for the confluent's RESTful API, the timeout here is to access and get response from the API.

`TimeSpan` is more readable and of course you can have milliseconds precision on `TimeSpan`.

For example, `TimeSpan.Parse("00:00:00.012")` will give you 12 ms timeout.